sfe.api/Controllers/VisitController.cs

```diff
@@ -26,9 +26,15 @@ public Visit Get(int id)
         }
 
         // GET api/<controller>/5
-        public IEnumerable<Visit> GetByUser(int userId)
+        public IEnumerable<Visit> GetByUser(int idUser)
```

Comment:
this is not a rule, but in my code usually I identify the parameters with a "p", something like "pIdUser" in order to detect where this parameter is used. This is old school but if you want you can change it